templates/todo/projects/csharp-sql/README.md

```diff
@@ -1,30 +1,18 @@
-# To Do Application with C# and Azure SQL on Azure App Service
+# ToDo Application with a C# API and Azure SQL Database
 
-A complete sample To Do application that demonstrates how to build an Azure solution using C#, Azure SQL for storage, and Azure Monitor for monitoring and logging.
+[![Open in Remote - Containers](https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/azure-samples/todo-csharp-cosmos-sql)
 
-## To Do Application
+A complete ToDo application that includes everything you need to build, deploy, and monitor an Azure solution. This application uses the Azure Developer CLI (azd) to get you up and running on Azure quickly, React.js for the Web application, C# for the API, Azure SQL database for storage, and Azure Monitor for monitoring and logging. It includes application code, tools, and pipelines that serve as a foundation from which you can build upon and customize when creating your own solutions.
 
-Here's the web frontend of this To Do application:
+Let's jump in and get the ToDo app up and running in Azure. When you are finished, you will have a fully functional web app deployed on Azure. In later steps, you'll see how to setup a pipeline and monitor the application.
 
-!["To Do Web App"](assets/web.png)
+img src="assets/web.png" width="75%" alt="Screenshot of deployed ToDo app">
 
-## Azure Resources
-
-This application is comprised of the following Azure resources:
-
-- Azure App Services - To host the To Do app website and API.
-- Azure SQL - To store the To Do data in a database.
-- Azure Monitor - A cloud instrumentation service to help you monitor your application.
-
-## Setup
-
-To run this project, you first need to setup your local development environment and get the code.
-
-> NOTE: Support for GitHub Codespaces and VS Code Remote Containers (DevContainers) is coming soon.
+<sup>Screenshot of the deployed ToDo app</sup>
 
 ### Prerequisites
 
-The following prerequisites are required to use this application.  Please ensure that you have them all installed locally.
+The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
 
 - [Azure Developer CLI](https://aka.ms/azure-dev/install)
```

Comment:
I really think that we should link to install instructions for the CLI (as opposed to including them inline here) so that we can reference a single source of truth and ensure that things don't drift or get stale over time.

IMO, this potential concern could be magnified if someone forks the template, makes it their own and then doesn't update the README, meaning that an end user could be pointed at out-of-date install instructions for the CLI and have a poor onboarding experience. I think it's pretty common to link out to docs for pre-reqs for repos as opposed to in-lining them.